backend/LexBoxApi/Services/ProjectService.cs

```diff
@@ -269,6 +271,36 @@ public async Task ResetLexEntryCount(string projectCode)
         }
     }
 
+    public async Task<DirectoryInfo?> ExtractLdmlZip(Project project, string destRoot, CancellationToken token = default)
+    {
+        if (project.Type != ProjectType.FLEx) return null;
+        var zip = await hgService.GetLdmlZip(project.Code, token);
```

Comment:
should be a using statement to dispose of the zip once this method returns

Reply:
Done in commit 3910173f.

Reply:
Done in commit bcc51cbc.